test/resources/v3/orders.test.js

```diff
@@ -1,9 +1,9 @@
 const RO = require('../../..');
+const { mockConfig } = require('../../test-helpers/mock-config');
 
 describe('v3 orders', () => {
   beforeAll(() => {
-    RO.config.set('apiVersion', 'v3');
-    RO.config.set('verbose', false);
+    RO.config.init(mockConfig({ apiVersion: 'v3', piiServerUrl: null, verbose: false }));
```

Comment:
Question: Why is `piiServerUrl: null` declared here?

Reply:
Ah, because it's set in `mockConfig`. Can we create a `mockPiiConfig`, please, so we have one each for the standard and PII setups? This would decouple these kinds of tests from PII-related tests — and likely eliminate the risk of bugs.